packages/pwa-kit-runtime/src/utils/morgan-stream.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const Writable = require('stream').Writable
+/**
+ * @private
+ * A stream that uses console.log to write log lines
+ */
+class ConsoleLogStream extends Writable {
+    write(line) {
+        console.log(line)
+    }
```

Comment:
Is this a new method extended from `Writable` or is this overloading `write` from the base class?

Reply:
This method will overload the write from the base class.

Reply:
In that case, I just want to be sure that we're confident we don't need to call `super()` for any side effects that are critical

Reply:
Yea, I don't think we need to since we solely want to console.log the log lines out, and there is no need to call super for it to work. I've tested the output log on one of MRT env, and it is working as expected